packages/v4/src/content/design-guidelines/components/context-selector/context-selector.md

```diff
@@ -2,15 +2,59 @@
 id: Context selector
 section: components
 ---
-A **context selector** can be used in addition to global navigation when the data or resources you show in the interface need to change depending on the users’ context.
+A **context selector** can be used in addition to global navigation when the data or resources you show in the interface need to change depending on the user’s context.
 
 ## Usage
 
-Context selectors enable a user to switch between different application contexts, like administrator and regular user view, for example.
+Context selectors enable a user to switch between different application contexts. They can be placed at many levels of an application’s information architecture and may have different behavior expectations based on the level of the IA it is placed in. They can be placed in a page masthead, at the top of vertical navigations, or at the top of a page, depending on your website or product's navigation and preference.
```

Comment:
In the first sentence, can we strike the word "IA?" May not be a common abbreviation familiar to all users. Maybe just say "...based on the level it is placed in."